sql/moz-fx-data-shared-prod/udf/organic_vs_paid_desktop/udf.sql

```diff
@@ -0,0 +1,37 @@
+CREATE OR REPLACE FUNCTION udf.organic_vs_paid_desktop(medium STRING, adjust_network STRING)
+RETURNS STRING AS (
+  CASE
+    WHEN LOWER(medium) IN (
+        'paidsearch',
+        'paiddisplay',
+        'cpc',
+        'email',
+        'display',
+        'paidsocial',
+        'paidvideo',
+        'social',
+        'ppc',
+        'banner',
+        'ad-unit',
+        'newsletter'
+      )
+      OR LOWER(adjust_network) IN (
```

Comment:
We don't have adjust_network on desktop tables

Reply:
Good to know, thanks! I removed it from the UDF - please re-review when you get time